app/views/step_by_step_pages/show.html.erb

```diff
@@ -18,6 +18,13 @@
 
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
+    <% unless @step_by_step_page.steps_have_content? %>
```

Comment:
Where has this moved from? Can you update the commit note to say where it was before?

Reply:
Also, I think it would be better to use `can_be_published?` as the guard clause, because that's what's being checked.